doc/user_guide/statisticalAnalysisExample.tex

```diff
@@ -192,3 +192,4 @@ \section{Statistical Analysis through RAVEN}
 \end{table}
 \end{landscape}
 
+\input{statisticsToDataObject.tex}
```

Comment:
Just to double check, is this supposed to be part of the merge?

Reply:
It was not originally intended to, but there's not a better merge for it to go in with, and the example should be added.  I think the TeX file is there, too.

Reply:
Okay, thanks.